easee/__main__.py

```diff
@@ -16,11 +16,17 @@ def parse_arguments():
     parser = argparse.ArgumentParser(description="Read data from your Easee EV installation")
     parser.add_argument("-u", "--username", help="Username", required=True)
     parser.add_argument("-p", "--password", help="Password", required=True)
-    parser.add_argument("-c", "--chargers", help="Get chargers information", action="store_true",)
+    parser.add_argument(
```

Comment:
to lint in the future use `make lint` before checking in so we have the same linting